packages/riipen-ui/src/components/TableRow.jsx

```diff
@@ -25,23 +30,48 @@ class TableRow extends React.Component {
     /**
      * Whether to render the border for the row
      */
-    border: PropTypes.bool
+    border: PropTypes.bool,
+
+    /**
+     * Function to call when we mouseover the row
+     */
+    onMouseOver: PropTypes.func,
+
+    /**
+     * Function to call when the mouse exits the row
+     */
+    onMouseOut: PropTypes.func
   };
 
   static defaultProps = {
-    border: true
+    border: true,
+    forceHover: false
   };
 
   static contextType = ThemeContext;
 
   render() {
-    const { children, classes, hover, border } = this.props;
+    const {
+      children,
+      classes,
+      hover,
+      forceHover,
+      border,
+      onMouseOver,
+      onMouseOut
+    } = this.props;
 
     const theme = this.context;
 
     return (
       <React.Fragment>
-        <tr className={clsx(classes)}>{children}</tr>
+        <tr
+          className={clsx(classes, forceHover && "forceHover")}
```

Comment:
dont quote me on this, but I believe you can replace `forceHover && "forceHover"` with `{ forceHover }`